dist/router.es5.js

```diff
@@ -148,6 +148,7 @@ function routerViewPortDirective($animate, $compile, $controller, $templateReque
     }, viewPortName);
   }
 }
+routerViewPortDirective.$inject = ["$animate", "$compile", "$controller", "$templateRequest", "$rootScope", "$location", "$componentLoader", "$router"];
```

Comment:
Generated by ng-annotate. This is the only possible annotation since the source code uses named functions.